src/main/java/nonapi/io/github/classgraph/classloaderhandler/QuarkusClassLoaderHandler.java

```diff
@@ -127,6 +127,13 @@ private static void findClasspathOrderForQuarkusClassloader(final ClassLoader cl
             } else if ("io.quarkus.bootstrap.classloading.DirectoryClassPathElement".equals(elementClassName)) {
                 classpathOrder.addClasspathEntry(ReflectionUtils.getFieldVal(false, element, "root"), classLoader,
                         scanSpec, log);
+            } else {
+                Object rootPath = ReflectionUtils.invokeMethod(false, element, "getRoot");
+                if (rootPath instanceof Path) {
+                    classpathOrder.addClasspathEntry(ReflectionUtils.invokeMethod(false, element, "getRoot"),
```

Comment:
I'll merge this, but why did you invoke this method twice? Why not just use the value of `rootPath`?

Reply:
Ooops. That was an oversight. You fixed this, didn't you? Thank you for that!

Reply:
I did, no problem!